src/Page/Navbar.elm

```diff
@@ -171,7 +171,7 @@ view model =
             [ Navbar.itemLink [href "#"] [ text "Item 1"]
             , Navbar.itemLink [href "#"] [ text "Item 2"]
             ]
-        |> Navbar.view state.basicState
```

Comment:
This is consistent not with the model in the comments but the model in the code (cf. lines 22-25)